Package.swift

```diff
@@ -0,0 +1,28 @@
+// swift-tools-version:5.1
```

Comment:
Actually, I think this should be 5.0. I made a mistake in README and put 5.1 badge instead of 5.0, but the library does not use any Swift 5.1 only features. Cocoapods podspec also declares 5.0 as minimum version. 
So to sum up:
- we can lower this to 5.0
- badge in README should be fixed and replaced with 5.0

Reply:
Done in 8a95bee1b9b139a02bc2942a230be5c78ffc4ddf and 43980cd812e9227a5fa1155eff1c1213e131a253